libr/core/anal.c

```diff
@@ -1445,32 +1342,29 @@ R_API int r_core_anal_graph(RCore *core, ut64 addr, int opts) {
 			" fontname=\"%s\" fontsize=\"8\"];\n", font);
 	if (is_json)
 		r_cons_printf ("[");
-	//r_cons_flush ();
-#define inrange(x,f) ((x>=f->addr)&&(x<(f->addr+f->size)))
 	r_list_foreach (core->anal->fcns, iter, fcni) {
 		if (fcni->type & (R_ANAL_FCN_TYPE_SYM | R_ANAL_FCN_TYPE_FCN)
-				&& (addr == 0 || inrange (addr, fcni))) {
```

Comment:
ok so for this PR i will keep using the macro in_function (i have merged inrange and infun because they were the same). Later on when I will review and refactor better this file I will probably create a API to check if an address is inside a function or not. How does it sound?